packages/fxa-auth-server/lib/senders/partials/base/base.html

```diff
@@ -27,7 +27,9 @@
   <td valign="top">
     <p style="font-family: sans-serif; font-weight: normal; margin: 0; text-align: center; color: #737373; font-size: 11px; line-height: 18px; width: 310px !important; word-wrap:break-word">Mozilla. 331 E Evelyn Ave, Mountain View, CA 94041
     <br />
-    <a href="{{{privacyUrl}}}" style="color: #0a84ff; text-decoration: none; font-family: sans-serif; font-size: 11px; line-height: 18px;">{{t "Mozilla Privacy Policy" }}</a></p>
+    <a href="{{{privacyUrl}}}" style="color: #0a84ff; text-decoration: none; font-family: sans-serif; font-size: 11px; line-height: 18px;">{{t "Mozilla Privacy Policy" }}</a>
+    <br />
+    <a href="https://www.mozilla.org/about/legal/terms/services/" style="color: #0a84ff; text-decoration: none; font-family: sans-serif; font-size: 11px; line-height: 18px;">{{t "Firefox Cloud Terms of Service"}}</a></p>
```

Comment:
I think updates here are supposed to cause all of the actual templates to be re-created.